tests/test_printers.py

```diff
@@ -17,6 +17,14 @@
 from substra.cli import printers
 
 
+class GlobalConf:
+    profile = None
+    config = None
+    user = None
+    verbose = None
+    expand = None
```

Comment:
Instead of re-defining this object, you could  just import it from interface.py.

Reply:
That's true, I'll do that!